src/Calendar.Plugin.Sample/SampleApp/MainPageViewModel.cs

```diff
@@ -13,6 +13,7 @@ namespace SampleApp
     public class MainPageViewModel : INotifyPropertyChanged
     {
         private CultureInfo _culture = CultureInfo.InvariantCulture;
+        public Action<DateTime> DayTappedAction { get; set; }
```

Comment:
Usually in XF we never use Action-s in bindable properties, I would prefer if this was a ICommand instead
Also I would rename the property from `DayTapped` to `DayTappedCommand`

Reply:
Made the requested changes.